recipes/ruby/build.sh

```diff
@@ -0,0 +1,9 @@
+set -e
+set -x
+
+./configure --prefix=$PREFIX --disable-install-doc --enable-load-relative \
+  --enable-shared --with-openssl-dir=$PREFIX --with-readline-dir=$PREFIX \
+  --with-tcl-dir=$PREFIX --with-tk-dir=$PREFIX --with-libyaml-dir=$PREFIX \
+  --with-zlib-dir=$PREFIX
```

Comment:
Just double checked, there is no with gmp dir, only `--without-gmp`.  

I'll quote them